ExomeDepth/make_BEDdetail.py

```diff
@@ -49,6 +50,12 @@ def slice_vcf(args, merge_dic):
     event_dic = {}
     children = 0
     parents = 0
+
+    # Random shuffle to truly select random samples in subset (function calculate_statistics)
+    # Random shuffle has not influence on general statistics
+    if not args.random_off:
+        random.shuffle(vcf_files)
```

Comment:
I am not sure about this one, but should it be something like
`vcf_files = random.shuffle(vcf_files)` or will vcf_files update automatically?

Reply:
Nope:

`>>> import random
>>> testlist = ["test2", "test3", "test1"]
>>> random.shuffle(testlist)
>>> testlist
['test1', 'test3', 'test2']
>>> random.shuffle(testlist)
>>> testlist
['test2', 'test3', 'test1']
>>> testlist = random.shuffle(testlist)
>>> testlist
>>>`

random sort is within the list